src/components/TodoItem/TodoItem.js

```diff
@@ -0,0 +1,115 @@
+import React, { Component } from 'react';
+import cx from 'classnames';
+import PropTypes from 'prop-types';
+
+export class TodoItem extends Component {
+  state = {
+    edtiTitle: this.props.todo.task,
+    placeholder: '',
+  };
+
+  markCompleted = (event) => {
+    const { checked, id } = event.target;
+    const { changeStatus } = this.props;
+
+    changeStatus(id, checked);
+  }
+
+  deleteTask = (event) => {
+    const { name } = event.target;
+    const { removeTask } = this.props;
+
+    removeTask(name);
+  }
+
+  handleEdit = (event) => {
```

Comment:
why do you create a separate method for this?
the same for other methods

Reply:
fix, delete unnecessary methods